tools/integration/lib/fetch.js

```diff
@@ -0,0 +1,24 @@
+// (c) Copyright 2024, SAP SE and ClearlyDefined contributors. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const fetch = (...args) => import('node-fetch').then(({ default: fetch }) => fetch(...args))
+// TODO: remove this once fetch is available in Node
```

Comment:
Fwiw, we could totally decide to run these tests with Node 21 right now and use native fetch 😇

Reply:
hmm... Node LTS is preferred.  It is also intentionally kept in sync with setup-node action in service, crawler for easier maintenance.